src/main/java/org/ow2/proactive/catalog/util/parser/WorkflowParser.java

```diff
@@ -258,7 +273,11 @@ private void handleElementWithMultipleValues(ImmutableList.Builder<KeyValueLabel
             }
         }, xmlStreamReader);
 
-        keyValueMapBuilder.add(new KeyValueLabelMetadataEntity(key[0], value[0], attributeLabel));
+        if (key[0] != null && value[0] != null) {
+            if (allowEmptyValues || (!value[0].isEmpty() && !key[0].isEmpty())) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the enclosing one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)